src/MSRIOGroup.hpp

```diff
@@ -209,6 +212,17 @@ namespace geopm
             std::shared_ptr<geopm_time_s> m_time_zero;
             std::shared_ptr<double> m_time_batch;
 
+            struct rdt_info
+            {
+                bool rdt_support;
+                uint32_t rmid_max;
```

Comment:
Currently rmid_max is not used, instead per cpu family bit ranges for RMID related registers are defined in the appropriate msr json file.